airflow/models/dag.py

```diff
@@ -73,7 +73,7 @@
 
 from airflow import settings, utils
 from airflow.api_internal.internal_api_call import internal_api_call
-from airflow.assets import Asset, AssetAlias, BaseAsset
```

Comment:
why we removed it here?

Reply:
Yeah, it removed by mistake when moving and import resolution, will update. :)